plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -1098,6 +1098,9 @@ private void testCreateTableLikeForFormat(IcebergFileFormat otherFormat)
         File tempDir = getDistributedQueryRunner().getCoordinator().getBaseDataDir().toFile();
         String tempDirPath = tempDir.toURI().toASCIIString() + randomTableSuffix();
 
+        // LIKE source INCLUDING PROPERTIES copies all the properties of the source table, including the `location`.
+        // For this reason the source and the copied table will share the same directory.
+        // This test does not drop intentionally the created tables to avoid affecting the source table or the information_schema.
```

Comment:
At the moment, dropping one of the tables breaks the drop procedure for the other tables.
Without specific logic to avoid mixing the content of the tables in the same directory I don't know how to perform the `DROP` statements.